great_expectations_provider/common/errors.py

```diff
@@ -0,0 +1,137 @@
+from __future__ import annotations
+
+from typing import TYPE_CHECKING, Any
+
+if TYPE_CHECKING:
+    from airflow.utils.context import Context
+    from great_expectations.checkpoint.checkpoint import CheckpointDescriptionDict
+
+
+def _extract_failed_types_from_expectations(expectations: Any) -> set[str]:
```

Comment:
Why `Any`?

Reply:
mypy is resolving the types from gx core as `Any | object`. i preferred being explicit about the `Any` type rather than casting.